src/manage_nsfs/health.js

```diff
@@ -519,13 +525,34 @@ async function is_bucket_storage_path_exists(fs_context, config_data, storage_pa
         } else if (err.code === 'EACCES' || (err.code === 'EPERM' && err.message === 'Operation not permitted')) {
             dbg.log1('Error:  Storage path should be accessible to account: ', storage_path);
             err_code = health_errors.ACCESS_DENIED.error_code;
+        } else if (err.code === 'BucketSetForbiddenBucketOwnerNotExists') {
```

Comment:
something is not clear here, BucketSetForbiddenBucketOwnerNotExists should be mapped MISSING_OWNER I meant to have INVALID_OWNER when the get_fs_context() threw an error.

Reply:
`BucketSetForbiddenBucketOwnerNotExists` thrown from method `get_bucket_owner_account_by_id()` we are catching it here